web-marketplace/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -85,6 +89,7 @@ const Dashboard = (): JSX.Element => {
   );
 
   const profileLink = getProfileLink(wallet?.address || activeAccount?.id);
+  const walletAddress = getWalletAddress({ activeAccount, wallet });
```

Comment:
What's the distinction between wallet and activeAccount?

Reply:
The `activeAccount` is the data of the logged-in account stored in db while `wallet` represents the wallet info we get from Keplr on the client side.
`getWalletAddress` looks for the wallet address in `activeAccount.addr` first then looks into `wallet?.address`.
This is to display the right address on the profile page and supports multiple types of users, whether it's web2 account with no address or WalletConnect users for which we can't have a logged-in account but still can access their wallet address and allow blockchain actions.